Tests/SentryTests/SentryViewHierarchyTests.swift

```diff
@@ -26,167 +34,185 @@ class SentryViewHierarchyTests: XCTestCase {
     }
 
     func test_Multiple_Window() {
-        let firstWindow = UIWindow(frame: CGRect(x: 0, y: 0, width: 10, height: 10))
-        let secondWindow = UIWindow(frame: CGRect(x: 0, y: 0, width: 10, height: 10))
+        if #available(iOS 14, *) {
```

Comment:
Im still testing, but it looks like the 13 also have the problem

Reply:
You might be able to do this in an `setUpWithError` override instead of inserting it at the beginning of each test method, which diffs every line due to indentation and could be forgotten for new test methods added in the future.

```swift
override func setUpWithError() throws {
	try super.setUpWithError()
	guard #available(iOS 14, *) else {
		throw XCTSkip("Skipping for iOS >14")
	}
}
```